src/main/java/mekanism/common/lib/distribution/SplitInfo.java

```diff
@@ -15,5 +16,9 @@ protected SplitInfo(int totalTargets) {
 
     public abstract TYPE getRemainderAmount();
```

Comment:
can we get some javadocs on these methods? I have zero idea how it all works without trying to trace it in an IDE